tools/archives/pyvo_integration/astronomical_archives.py

```diff
@@ -695,7 +696,7 @@ def _set_cone_query(self):
         ts = 'target_selection'
         con = 'cone_object_name'
 
-        search_radius = self._json_parameters[qs][qse]['radius']
+        search_radius = self._json_parameters[qs][qsl]['radius']
```

Comment:
that looks more like a linting fix ... do you want to bump the version number?

Reply:
Not totally sure what you mean by that?

The version number has already been bumped in the previous pull request (https://github.com/esg-epfl-apc/tools-astro/pull/40) that adds a new feature, this one is only there to fix some linting issues related to the previous pr

Reply:
I guess @bgruening is pointing out that this change, `qse` => `qsl`, does not look like a linting change.
In fact, `qse` is not defined, so it was a typo, right @francoismg ?

Then, the question is if we need to update tool version. For this fix. Since the previous version did not actually get published to toolsched, it does not seem necessary to bump the version with this bug and lint fix, right @bgruening ?f

Reply:
I guess @bgruening is pointing out that this change, `qse` => `qsl`, does not look like a linting change.
In fact, `qse` is not defined, so it was a typo, right @francoismg ?

Then, the question is if we need to update tool version. For this fix. Since the previous version did not actually get published to toolsched, it does not seem necessary to bump the version with this bug and lint fix, right @bgruening ?f

Reply:
If its not published you don't need to bump the version, that is correct. I missed the fact that it was not pushed to the TS.